docs/azure/sdk/snippets/authentication/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿#region snippet_UseCredential
+using Azure.Core;
+using Azure.Identity;
+using Microsoft.Extensions.Azure;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    clientBuilder.AddBlobServiceClient(
+        new Uri("https://<account-name>.blob.core.windows.net"));
+    #region snippet_Dac
+    clientBuilder.UseCredential(new DefaultAzureCredential());
+    #endregion
+
+    #region snippet_DacExcludes
+    clientBuilder.UseCredential(new DefaultAzureCredential(
+        new DefaultAzureCredentialOptions
+        {
+            ExcludeEnvironmentCredential = true,
+            ExcludeWorkloadIdentityCredential = true,
+        }));
+    #endregion
+
+    #region snippet_Ctc
+    clientBuilder.UseCredential(new ChainedTokenCredential(
+        new ManagedIdentityCredential(),
+        new VisualStudioCredential()));
```

Comment:
All credentials in DAC support a parameterless default constructor.

Let's include a credential that requires parameters in its constructor in the sample, so we can demonstrate how to create a chain for such scenarios.

For example, we could use `ClientSecretCredential`, `CertificateCredential`, or `AzurePipelinesCredential`.